art-decisions/proposals/2023-03-08-automated-dependency-updates.md

```diff
@@ -0,0 +1,66 @@
+
+Overall, Renovate can help you save time and effort, improve the security, stability, and performance of your code, and customize the tool to fit your specific needs.
+
+## Renovate vs. Dependabot
+
+Both Renovate and Dependabot are excellent tools for automating dependency updates in your repositories. Here are some reasons why you might choose Renovate over Dependabot:
+
+1. More flexible configuration: Renovate offers more flexible configuration options than Dependabot, allowing you to fine-tune the update behavior to match your needs. Renovate's configuration options include things like update schedules, filtering rules, and package manager-specific options.
+
+2. Better update handling: Renovate can handle complex updates better than Dependabot, which is especially useful when you have dependencies with multiple release channels or non-standard versioning schemes.
+
+3. More supported package managers: Renovate supports a wider range of package managers than Dependabot, including Yarn, Pip, and Rubygems.
+
+4. More frequent updates: Renovate has a more frequent update schedule than Dependabot, which means you can get updates to the tool and its dependencies faster.
+
+5. Stronger community support: Renovate has a strong and active community of users and contributors who can provide support and contribute to the tool's development.
+
+That being said, Dependabot is still a great tool for automating dependency updates, and may be a better fit for your specific needs. Ultimately, the choice between Renovate and Dependabot depends on your specific use case and preferences.
+
+## Drawbacks
+
+While Renovate can be a powerful tool for automating dependency updates, there are a few potential drawbacks to consider:
+
+1. Pull request overload: Renovate can generate a large number of pull requests, especially if you have many repositories or dependencies. This can create a lot of noise in your pull request feed and may require additional effort to manage.
+
+2. False positives: Renovate may sometimes generate pull requests for updates that don't actually improve your dependencies or may introduce new issues. This can create additional work and may require manual intervention to resolve.
+
+3. Configurability complexity: While Renovate offers a lot of configuration options, this can also make it more complex to set up and manage than other dependency update tools.
+
+4. Limited control over updates: Renovate automates the update process, but you may not have full control over which updates are applied to your dependencies. This can be a concern if you need to ensure strict compatibility or stability with certain packages or versions.
+
+5. Potential conflicts with other automation: If you have other automation tools or processes in place that also manage dependency updates, Renovate may conflict with them and create additional work or confusion.
```

Comment:
Do _not_ run Dependabot and Renovate together, you'll end up with a mess of open PRs.

If you want to get `vulnerabilityAlerts` [^vulnerabilityAlerts] then you'll need to enable the "Dependency Graph" and "Dependabot Alerts". You still keep the "normal Dependabot" _disabled_ though!

[^vulnerabilityAlerts]: https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts